chore(kics): upgrade KICS to remove deprecated NodeJS version warning

alv-lop opened the pull request:
Signed-off-by: Alvaro Lopez Hernandez <alvaro.lopezhernandez@engineering.digital.dwp.gov.uk>

dan-hill2802 commented:
I'm not sure `platform_type` is used anymore (might be worth confirming in KICS docs). Might be worth enabling comments too
example:
```
      - name: KICS Github Action
        uses: checkmarx/kics-action@v1.5
        with:
          path: "repo"
          output_path: "results"
          exclude_queries: ${{ env.KICS_apt_version_pin }}
          fail_on: high,medium
          enable_comments: true
```

alv-lop commented:
@dan-hill2802 `platform_type` appears to be part of their docs and I don't see any reference to it being deprecated.

I've added `enable_comments: true` and also converted the queries to env variables so we more clearly know what they are doing, similar to how we do in https://github.com/dwp/github-action-kitchen-terraform

dan-hill2802 commented on .github/workflows/pr.yml:
```diff
@@ -33,14 +38,15 @@ jobs:
           output_path: 'results'
           platform_type: terraform
           fail_on: high,medium
-          exclude_queries: '8e94dced-9bcc-4203-8eb7-7e41202b2505,e592a0c5-5bdb-414c-9066-5dba7cdea370,7ebc9038-0bde-479a-acc4-6ed7b6758899,91f16d09-689e-4926-aca7-155157f634ed,a88baa34-e2ad-44ea-ad6f-8cac87bc7c71' # Excludes 'Auto Scaling Group With No Associated ELB', 'IAM Access Analyzer Undefined' queries, 'Vulnerable policy for the KMS Key' ECS Service without tasks running and excludes where ...
```
I like that you've moved these to envs, but could you add the comments to each env var as they are the vulnerability titles

alv-lop replied:
Yep sure. Added a comment with their query titles next to each env var.
